src/serializers/fields.rs

```diff
@@ -53,7 +53,7 @@ impl SerField {
         }
     }
 
-    pub fn get_key_py<'py>(&'py self, py: Python<'py>, extra: &Extra) -> &Bound<'py, PyAny> {
+    pub fn get_key_py<'py>(&'py self, py: Python<'py>, extra: &Extra) -> &'py Bound<'py, PyAny> {
```

Comment:
I think more correct is to drop the `&'py` constraint from `self` and let the lifetime elision apply on the return value too:

```suggestion
    pub fn get_key_py<'py>(&self, py: Python<'py>, extra: &Extra) -> &Bound<'py, PyAny> {
```